djstripe/models.py

```diff
@@ -106,7 +124,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(StripeObject):
     stripe_api_name = "Event"
-
+    objects = models.Manager()
```

Comment:
I think what I was going for was separation of concerns. Isolating Stripe API related bulk operations (such as finding something by its stripe id) in one place and having another place which does all the high level model operations of a manager.

The first would be mostly for the consumption of the model itself (not quite a private member, but nearly). The second, would be for the use of users of the model.

I don't think it's a bad idea, but I don't feel too strongly one way or the other. I'll leave it up to you.